llvm/CMakeLists.txt

```diff
@@ -286,6 +286,9 @@ if(LLVM_CCACHE_BUILD)
       endif()
       set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ${CCACHE_PROGRAM})
     else()
+      # Until a way to reliably configure ccache on Windows is found,
+      # disable precompiled headers for Windows + ccache builds
+      set(CMAKE_DISABLE_PRECOMPILE_HEADERS "ON")
```

Comment:
Consider emitting a message informing the user if their `CMAKE_DISABLE_PRECOMPILE_HEADERS=OFF` setting was overwritten.

Reply:
Good suggestion, thank you! Will do.